h/api/models/document.py

```diff
@@ -37,7 +37,7 @@ def title(self):
         titles = [m.value for m in self.meta if m.type == 'title']
 
         if titles:
-            return titles[0]
+            return titles[0][0]
```

Comment:
Something has clearly gone wrong here. The value of a single meta instance should not be an array, I don't think.